modules/ROOT/pages/amazon/amazon-sns-connector.adoc

```diff
@@ -41,12 +41,8 @@ Amazon SNS connector is compatible with:
 
 == Audience
 
-* Starting user
-+
-To create your Mule app, see xref:amazon/amazon-sns-connector-design-center.adoc[Design Center Configuration] or xref:amazon/amazon-sns-connector-studio.adoc[Studio Configuration]. The xref:amazon/amazon-sns-connector-examples.adoc[Examples] topic provides one or more use cases for using this connector.
-* Power user
-+
-Read the xref:amazon/amazon-sns-connector-xml-maven.adoc[XML and Maven Support]
+* Starting user -- To create your Mule app, see xref:amazon/amazon-sns-connector-design-center.adoc[Design Center Configuration] or xref:amazon/amazon-sns-connector-studio.adoc[Studio Configuration]. The xref:amazon/amazon-sns-connector-examples.adoc[Examples] topic provides one or more use cases for using this connector.
```

Comment:
```suggestion
* Starting user - To create your Mule app, see xref:amazon/amazon-sns-connector-design-center.adoc[Design Center Configuration] or xref:amazon/amazon-sns-connector-studio.adoc[Studio Configuration]. The xref:amazon/amazon-sns-connector-examples.adoc[Examples] topic provides one or more use cases for using this connector.
```

Reply:
I may have also made the wrong correction here. "Starting user" has two dashes, and "Power user" has one dash. I guessed that this one should change but maybe it's the other that should change.